### DISCUSSION TITLE
Development reload support

### DISCUSSION THREAD

**Discussion by repli2dev** [php/frankenphp#132] [Q&A] [answered]
If I did not miss anything I did not found how to set it up in development reload mode so that I do not need to restart the frankenphp container...

Is there a way for this?

**Comment by repli2dev** [comment, marked as answer]
Another test says it works (unless in worker mode), weird that it did not on larger project.

**Reply by dunglas** [reply]
Did you check your opcache settings? Especially, double-check that you did not disable timestamp validation.

**Reply by repli2dev** [reply]
Double checked and it was it! Thanks :-)
